src/Aspire.Dashboard/Components/Controls/EnvironmentVariables.razor.cs

```diff
@@ -9,13 +9,14 @@ namespace Aspire.Dashboard.Components.Controls;
 
 public partial class EnvironmentVariables
 {
-
     [Parameter, EditorRequired]
     public IEnumerable<EnvironmentVariableViewModel>? Items { get; set; }
 
     [Parameter]
     public bool ShowSpecOnlyToggle { get; set; }
 
+    private bool IsSpecOnlyToggleDisabled => Items?.Any(i => i.FromSpec == false) is false or null;
```

Comment:
My pattern matching knowledge is rusty... Does `is not true` work here (I can see the argument for being explicit about null even if it does work, mostly just curious)?

Reply:
Your intuition is right. They're logically equivalent.

I expected them to produce the same assembly, but it looks like there's a small difference: https://sharplab.io/#v2:EYLgtghglgdgNAFxBAzmAPgAQEwEYCwAUJgMwAEOZAwmQN5FmMXnAD2rANmQLK4AUbTgH4ywAJRkAvAD5RZKCjIAzCBxQBTMqwBOZGAFcOHANwMmpUey7dsAqyPFTZweYpisEZBNv3rThAF8gA==